### PR TITLE
Implement Navigation Setup

### PR DESCRIPTION
**Pull request by law-levi-san**

### CODE REVIEW

File: types/NavigationProps.tsx
Comment (Kicchu02) [inline types/NavigationProps.tsx:1]: I do not know what this file is doing, I am just trusting you now. Still, I would appreciate it if you would explain the working to me.

**Review by Kicchu02 — CHANGES_REQUESTED** [review]
image


Can you figure out a way to remove this header? IMO it looks better without those headers, according to our design style.

File: types/NavigationProps.tsx
Comment (law-levi-san) [reply types/NavigationProps.tsx:1]: I read that by creating a file, which is used to store the screen names would be helpful in navigation. By specifying the available screens in the 'StackParamList' it aids in navigating and switching between screens. This method is rather direct in javascript and there isn't any need to create a seperate file like this, but in typescript there is.

**Comment by law-levi-san** [comment]
> image
> 
> Can you figure out a way to remove this header? IMO it looks better without those headers, according to our design style.

I will try

**Comment by Kicchu02** [comment]
> I will try

Make sure the last comments reflect the current approach or state of the code. This means you are still trying, but you have already fixed it.

**Review by Kicchu02 — APPROVED** [review]

**Pull request merged**